### PR TITLE
Replace Skopeo with composeapp

### PR DESCRIPTION
**Pull request by kprosise**
Updated, with link to the composeapp repo.

QA: ran build and checked output in browser. Ran linkcheck locally. No issues to report.

No related tasks, small fix mentioned via internal channel.

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 2e02c29654af5a6c09136cd4abb3c09293270a26 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2734/docs/artifacts/html/index.html

**Review by vanmaegima — APPROVED** [review]

File: source/reference-manual/docker/restorable-apps.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Restorable Apps
 Restorable Apps are Compose Apps that can be restored in case of any Docker store damages, without needing to be re-downloaded from the Registry.
 This is achieved by a distinct way images can pulled and stored on a device.
 
-#. Images are pulled from registries by the `skopeo <https://github.com/containers/skopeo>`_ utility,
+#. Images are pulled from registries by the `composeapp <https://github.com/foundriesio/composeapp>`_ utility,
```
Comment (vanmaegima) [inline source/reference-manual/docker/restorable-apps.rst:9]: not sure if we should call it `composectl` instead (@mike-sul can comment)

Comment (mike-sul) [reply source/reference-manual/docker/restorable-apps.rst:9]: Yes, it should be `composectl`. The project/repo is called "composeapp" while the CLI utility is called `composectl` - it is one of the possible artifacts that the composeapp project can produce (future).

**Comment by doanac** [comment]
Docs for 80958c0deaef8689d6d00230e144f573fc5bc52c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2735/docs/artifacts/html/index.html

**Pull request merged**